test/functional/apps/home/_navigation.js

```diff
@@ -30,8 +29,18 @@ export default function ({ getService, getPageObjects }) {
   const fromTime = '2015-09-19 06:31:44.000';
   const toTime = '2015-09-23 18:31:44.000';
 
-  describe('Kibana browser back navigation should work', function describeIndexTests() {
+  const getBasePath = () => {
+    if (process.env.TEST_KIBANA_URL) {
+      const myURL = new URL(process.env.TEST_KIBANA_URL);
+      return `${myURL.protocol}//${myURL.host}`;
+    }
+    const protocol = process.env.TEST_KIBANA_PROTOCOL || 'http';
+    const host = process.env.TEST_KIBANA_HOST || 'localhost';
+    const port = process.env.TEST_KIBANA_PORT || '5620';
+    return `${protocol}://${host}:${port}`;
+  };
```

Comment:
Dear valued reviewer, this is the main change. adding a function for the used basePath and taking care of more possible env variables. there are some minor additional adaptions by my pal prettier